packages/mobile/src/screens/auth/components/line/line.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { Text, View } from '~/components/components';
+import { useCustomTheme, useTranslation } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const Line: FC = () => {
```

Comment:
The naming is quite confusing. Let's make this component more universal as we might use if in future.
Let's call it `Divider` and receive text as a prop so that we may use it without any text at all

Reply:
corrected